azerothcore/azerothcore-wotlk
# feat(Core/Commands): Character check bag && work

## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3158

## HOW TO TEST THE CHANGES:
.character check bank (Show your bank inventory)
.character check bag ([$target_player OR YOUR SELF] #bagSlot 1 - 4)
character check work ([$target_player OR YOUR SELF] Show known professions list for selected player)

## Target branch(es):
